doc/man/cockpit.conf.xml

```diff
@@ -102,9 +102,15 @@ ForwardedForHeader = X-Forwarded-For
         <term><option>LoginTo</option></term>
         <listitem>
           <para>When set to <literal>true</literal> the <emphasis>Connect to</emphasis> option
-            on the login screen is visible and allows logging into another server. If this
-            option is not specified then it will be automatically detected based on whether
-            the <command>cockpit-ssh</command> process is available or not.</para>
+          on the login screen is visible and allows logging into another server. When set to
```

Comment:
The indentation change makes this patch hard to read.

I wonder if we should have the default just be False.  It's kinda dangerous, potentially unexpected, and `cockpit-ssh` is very easy to have installed.

It might break people's working configs on upgrade, but it would be easy to advise how to fix it.

Reply:
Err right, that wasn't intended. vim's auto-spacing has an eternal fight with the weird indentation of our doc files. I fixed this.

Changing the default is a much harder decision. Connecting to a remote machine is a rather useful and important feature of cockpit, and just changing it on upgrades breaks existing setups too hard IMHO. Also, in some sense we even *want* people to do this "remote host" thing -- it's really not encouraged to put cockpit-ws on production servers, but always go through some bastion host. cockpit-desktop/Cockpit Client do exactly that (of course these can get an explicit config). `LoginTo` also interacts with `RequireHost=true` -- if that is set, then `LoginTo=false` does not make sense. So if it's unset, then it should be implicitly `true` with `RequireHost=true`.

In other words, this is a mine field -- at some point we need to analyze carefully what makes sense here, and probably clean up this non-orthogonal configuration.